.travis.yml

```diff
@@ -66,25 +72,16 @@ matrix:
 
 install:
   - |
-    if [ "$SOM" != "spec" ]
-    then
-      pip install pyyaml
-      git clone --depth 1 $REPO som-vm
-    fi
+    pip install pyyaml
+    git clone --depth 1 $REPO som-vm
 
 # command to run tests
 script:
   - |
-    set -e
-    if [ "$SOM" = "spec" ]
-    then
-      cd specification
-      make test
-    else
-      ./TestSuite/BasicInterpreterTests/number-of-tests.sh
-      export ST_DIR=`pwd`/Smalltalk
-      cd som-vm
-      git --no-pager log -n 1
-      eval $BUILD
-      $SOM -cp ../Smalltalk ../TestSuite/TestHarness.som
-    fi
+    ./TestSuite/BasicInterpreterTests/number-of-tests.sh
+    export ST_DIR=`pwd`/Smalltalk
+    cd som-vm
+    git --no-pager log -n 1
+    eval $BUILD
+    $SOM -cp ../Smalltalk ../TestSuite/TestHarness.som
+    $SOM -cp ../Smalltalk:../TestSuite:../SomSom/src/compiler  ../SomSom/tests/SomSomTests.som
```

Comment:
I expect the CI to go red because of those tests, except for som-java once it will be updated.
Should we keep it red for now / allow failure for the others ?